src/fieri/app/controllers/fieri/jobs_controller.rb

```diff
@@ -20,5 +22,10 @@ def job_params
       end
       params
     end
+
+    def cookbook_version_response
+      uri = URI.parse("#{ENV['FIERI_SUPERMARKET_ENDPOINT']}/api/v1/cookbooks/#{params['cookbook_name']}/versions/#{params['cookbook_version']}")
+      Net::HTTP.get(uri)
+    end
```

Comment:
I would expect this lookup to occur within the LicenseWorker job being scheduled by hitting this endpoint, not by the endpoint itself prior to job scheduling.